dan_layer/template_lib/src/models/resource.rs

```diff
@@ -20,6 +20,8 @@
 //  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+use std::str::FromStr;
```

Comment:
Need to ensure that template_lib supports no_std
```suggestion
use tari_template_abi::rust::str::FromStr;
```

Reply:
Resolved